Specification/BrAPI-Schema/BrAPI-Phenotyping/ObservationUnitLevelRelationship.json

```diff
@@ -2,45 +2,21 @@
+            "allOf": [
+                {
+                    "$ref": "../BrAPI-Phenotyping/ObservationUnitLevel.json#/$defs/ObservationUnitLevel"
                 },
-                "levelName": {
-                    "description": "A name for this level \n\n**Standard Level Names: study, field, entry, rep, block, sub-block, plot, sub-plot, plant, pot, sample** \n\nFor more information on Observation Levels, please review the <a target=\"_blank\" href=\"https://wiki.brapi.org/index.php/Observation_Levels\">Observation Levels documentation</a>. ",
-                    "type": [
-                        "null",
-                        "string"
-                    ]
-                },
-                "levelOrder": {
-                    "description": "`levelOrder` defines where that level exists in the hierarchy of levels. `levelOrder`'s lower numbers \nare at the top of the hierarchy (ie field -> 1) and higher numbers are at the bottom of the hierarchy (ie plant -> 9). \n\nFor more information on Observation Levels, please review the <a target=\"_blank\" href=\"https://wiki.brapi.org/index.php/Observation_Levels\">Observation Levels documentation</a>. ",
-                    "type": [
-                        "null",
-                        "integer"
-                    ]
-                },
-                "observationUnitDbId": {
-                    "description": "The ID which uniquely identifies an observation unit\n<br/>If this level has on ObservationUnit associated with it, record the observationUnitDbId here. This is intended to construct a strict hierarchy of observationUnits. \n<br/>If there is no ObservationUnit associated with this level, this field can set to NULL or omitted from the response.",
-                    "type": [
-                        "null",
-                        "string"
-                    ]
-                },
-				"observationUnitLevelRelationshipDbId": {
-                    "description": "The ID which uniquely identifies a observaton unit level relationship",
-                    "type": [
-                        "null",
-                        "string"
-                    ]
+                {
+                    "type": "object",
+                    "properties": {
+                        "observationUnit": {
+                            "description": "The an observation unit\n<br/>If this level has on ObservationUnit associated with it, record the observationUnitDbId here. This is intended to construct a strict hierarchy of observationUnits. \n<br/>If there is no ObservationUnit associated with this level, this field can set to NULL or omitted from the response.",
+                            "$ref": "../BrAPI-Phenotyping/ObservationUnit.json#/$defs/ObservationUnit",
+                            "relationshipType": "one-to-one"
+                        }
+                    }
                 }
-            },
-            "title": "ObservationUnitLevelRelationship",
```

Comment:
No title  nor type for the model?

Reply:
no type is needed because it is an 'allof' schema, but it could have a title, although it is option can defaults the to name. I did not put titles in any of the 'allof' schemas.